src/main/java/org/openrewrite/staticanalysis/AnnotateNullableMethods.java

```diff
@@ -0,0 +1,181 @@
+
+        static AtomicBoolean find(J subtree) {
+            return new FindNullableReturnStatements().reduce(subtree, new AtomicBoolean());
+        }
+
+        private static List<MethodMatcher> getMatchersKnownNullableMethods() {
+            List<MethodMatcher> matchers = new ArrayList<>();
+
+            matchers.add(new MethodMatcher("java.util.Map compute(..)"));
+            matchers.add(new MethodMatcher("java.util.Map computeIfAbsent(..)"));
+            matchers.add(new MethodMatcher("java.util.Map computeIfPresent(..)"));
+            matchers.add(new MethodMatcher("java.util.Map get(..)"));
+            matchers.add(new MethodMatcher("java.util.Map merge(..)"));
+            matchers.add(new MethodMatcher("java.util.Map put(..)"));
+            matchers.add(new MethodMatcher("java.util.Map putIfAbsent(..)"));
+
+            matchers.add(new MethodMatcher("java.util.Queue poll(..)"));
+            matchers.add(new MethodMatcher("java.util.Queue peek(..)"));
+
+            matchers.add(new MethodMatcher("java.util.Deque peekFirst(..)"));
+            matchers.add(new MethodMatcher("java.util.Deque pollFirst(..)"));
+            matchers.add(new MethodMatcher("java.util.Deque peekLast(..)"));
+
+            matchers.add(new MethodMatcher("java.util.NavigableSet lower(..)"));
+            matchers.add(new MethodMatcher("java.util.NavigableSet floor(..)"));
+            matchers.add(new MethodMatcher("java.util.NavigableSet ceiling(..)"));
+            matchers.add(new MethodMatcher("java.util.NavigableSet higher(..)"));
+            matchers.add(new MethodMatcher("java.util.NavigableSet pollFirst(..)"));
+            matchers.add(new MethodMatcher("java.util.NavigableSet pollLast(..)"));
+
+            matchers.add(new MethodMatcher("java.util.NavigableMap lowerEntry(..)"));
+            matchers.add(new MethodMatcher("java.util.NavigableMap floorEntry(..)"));
+            matchers.add(new MethodMatcher("java.util.NavigableMap ceilingEntry(..)"));
+            matchers.add(new MethodMatcher("java.util.NavigableMap higherEntry(..)"));
+            matchers.add(new MethodMatcher("java.util.NavigableMap lowerKey(..)"));
+            matchers.add(new MethodMatcher("java.util.NavigableMap floorKey(..)"));
+            matchers.add(new MethodMatcher("java.util.NavigableMap ceilingKey(..)"));
+            matchers.add(new MethodMatcher("java.util.NavigableMap higherKey(..)"));
+            matchers.add(new MethodMatcher("java.util.NavigableMap firstEntry(..)"));
+            matchers.add(new MethodMatcher("java.util.NavigableMap lastEntry(..)"));
+            matchers.add(new MethodMatcher("java.util.NavigableMap pollFirstEntry(..)"));
+            matchers.add(new MethodMatcher("java.util.NavigableMap pollLastEntry(..)"));
+
+            matchers.add(new MethodMatcher("java.util.Spliterator trySplit(..)"));
+
+            return Collections.unmodifiableList(matchers);
+        }
```

Comment:
Please reduce boilerplate by using: https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/util/List.html#of(E...)
```suggestion
    private static List<MethodMatcher> getMatchersKnownNullableMethods() {
        return List.of(
                new MethodMatcher("java.util.Map compute(..)"),
                new MethodMatcher("java.util.Map computeIfAbsent(..)"),
                new MethodMatcher("java.util.Map computeIfPresent(..)"),
                new MethodMatcher("java.util.Map get(..)"),
                new MethodMatcher("java.util.Map merge(..)"),
                new MethodMatcher("java.util.Map put(..)"),
                new MethodMatcher("java.util.Map putIfAbsent(..)"),
                new MethodMatcher("java.util.Queue poll(..)"),
                new MethodMatcher("java.util.Queue peek(..)"),
                new MethodMatcher("java.util.Deque peekFirst(..)"),
                new MethodMatcher("java.util.Deque pollFirst(..)"),
                new MethodMatcher("java.util.Deque peekLast(..)"),
                new MethodMatcher("java.util.NavigableSet lower(..)"),
                new MethodMatcher("java.util.NavigableSet floor(..)"),
                new MethodMatcher("java.util.NavigableSet ceiling(..)"),
                new MethodMatcher("java.util.NavigableSet higher(..)"),
                new MethodMatcher("java.util.NavigableSet pollFirst(..)"),
                new MethodMatcher("java.util.NavigableSet pollLast(..)"),
                new MethodMatcher("java.util.NavigableMap lowerEntry(..)"),
                new MethodMatcher("java.util.NavigableMap floorEntry(..)"),
                new MethodMatcher("java.util.NavigableMap ceilingEntry(..)"),
                new MethodMatcher("java.util.NavigableMap higherEntry(..)"),
                new MethodMatcher("java.util.NavigableMap lowerKey(..)"),
                new MethodMatcher("java.util.NavigableMap floorKey(..)"),
                new MethodMatcher("java.util.NavigableMap ceilingKey(..)"),
                new MethodMatcher("java.util.NavigableMap higherKey(..)"),
                new MethodMatcher("java.util.NavigableMap firstEntry(..)"),
                new MethodMatcher("java.util.NavigableMap lastEntry(..)"),
                new MethodMatcher("java.util.NavigableMap pollFirstEntry(..)"),
                new MethodMatcher("java.util.NavigableMap pollLastEntry(..)"),
                new MethodMatcher("java.util.Spliterator trySplit(..)")
        );
    }
```

Reply:
We don't yet support `List.of` as we remain compatible with Java 8. `Arrays.asList` could work though.

Reply:
Changed to Arrays.off